src/Backend/Modules/Pages/Domain/Page/Page.php

```diff
@@ -2,23 +2,40 @@
 
 namespace Backend\Modules\Pages\Domain\Page;
 
-use Backend\Core\Engine\Model as BackendModel;
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroup;
+use Backend\Modules\MediaLibrary\Domain\MediaItem\MediaItem;
+use Backend\Modules\Pages\Domain\PageBlock\PageBlock;
 use Common\Doctrine\Entity\Meta;
+use Common\Locale;
 use DateTime;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
 use Doctrine\ORM\Mapping as ORM;
+use Frontend\Core\Engine\Navigation;
 
 /**
  * @ORM\Entity(repositoryClass="Backend\Modules\Pages\Domain\Page\PageRepository")
  * @ORM\Table(
  *     name="PagesPage",
  *     indexes={
- *      @ORM\Index(name="idx_id_status_hidden_language", columns={"id", "status", "language"})
+ *      @ORM\Index(name="idx_id_status_hidden_locale", columns={"id", "status", "locale"})
  *     }
  * )
  * @ORM\HasLifecycleCallbacks()
  */
 class Page
 {
+    public const NO_PARENT_PAGE_ID = 0;
+    public const HOME_PAGE_ID = 1;
+    public const ERROR_PAGE_ID = 404;
+    public const TOP_LEVEL_IDS = [
+        self::NO_PARENT_PAGE_ID,
+        self::HOME_PAGE_ID,
+        2,
```

Comment:
What are these?

Reply:
some pages that act as top level domains

Reply:
Could you add a comment stating what pages these refer to?

Reply:
fixed in 58a7d60ef